builtin/game/privileges.lua

```diff
@@ -74,15 +74,15 @@ core.register_privilege("password", {
 	give_to_admin = true,
 })
 core.register_privilege("fly", {
-	description = "Can fly using the free_move mode",
+	description = "Can use fly mode",
 	give_to_singleplayer = false,
 })
 core.register_privilege("fast", {
-	description = "Can walk fast using the fast_move mode",
+	description = "Can use fast mode",
 	give_to_singleplayer = false,
 })
 core.register_privilege("noclip", {
-	description = "Can fly through walls",
+	description = "Can fly through solid blocks using the noclip mode",
```

Comment:
- "the" for consistency.

Reply:
I agree, also, 'nodes' not blocks. Many nodes are solid but are not 'blocks'.

Reply:
And 'blocks' in MT often refers to 'mapblocks'.

Reply:
or `Can disable collisions in fly mode`